charlatan/file_format.py

```diff
@@ -73,6 +73,19 @@ def get_now():
 
         return get_now
 
+    def epoch_now_in_ms_constructor(loader, node):
+        """Return a function that returns the current epoch in milliseconds.
+
+        :rtype: int
+        """
+        delta = get_timedelta(loader.construct_scalar(node))
+
+        def get_now():
+            return int(1000 * datetime_to_epoch_timestamp(
+                datetime.datetime.utcnow() + delta))
```

Comment:
shouldn't you add the delta after the conversion to ms? i think the yaml files will be harder to read if the delta is in seconds for a field that is in ms.